applications/ehr_query_llm/fhir/README.md

```diff
@@ -29,14 +29,14 @@ There are several ways to build and run this application and package it as a Hol
 
 It is further expected that you have read the [HoloHub README](../../../README.md), have cloned the HoloHub repository to your local system, and the current working directory is the HoloHub root, `holohub`.
 
-**_Note_**:  
```

Comment:
Thanks @wyli for this PR. Just a quick note: the tailing double spaces at the end of a line in markdown is similar to <br>. Here and elsewhere, could please bring them back? Thanks